internal/controller/validator.go

```diff
@@ -56,11 +55,32 @@ func (r *DataProtectionApplicationReconciler) ValidateDataProtectionCR(log logr.
 		return validVsl, err
 	}
 
+	// ENSURE UPGRADES --------------------------------------------------------
 	// check for VSM/Volsync DataMover (OADP 1.2 or below) syntax
 	if r.dpa.Spec.Features != nil && r.dpa.Spec.Features.DataMover != nil {
 		return false, errors.New("Delete vsm from spec.configuration.velero.defaultPlugins and dataMover object from spec.features. Use Velero Built-in Data Mover instead")
 	}
 
+	// check for ResticConfig (OADP 1.4 or below) syntax
+	if r.dpa.Spec.Configuration.Restic != nil {
+		return false, errors.New("Delete restic object from spec.configuration, use spec.configuration.nodeAgent instead")
```

Comment:
`s/object/spec`

Reply:
not sold on this. Changing to spec would not pass sense that restic is an CR instead of part of DPA spec?